docs/repositories-configure/integrations/github-integration.md

```diff
@@ -48,3 +50,20 @@ Adds annotations on the lines of the pull request where Codacy finds new issues.
 Shows an overall view of the changes in the pull request, including new issues and metrics such as complexity and duplication. To enable this option, you must enable **Status checks** first.
 
 ![Pull request summary on GitHub](images/github-integration-pr-summary.png)
+
+### Suggest fixes
+
+Adds comments on the lines of the pull request where Codacy finds new issues with suggestions on how to fix the issues. Codacy doesn't apply any changes automatically. To apply the changes, manually review and accept the suggestions.
```

Comment:
Is it worth mentioning that these can be accepted individually in the "conversation" tab, or in bulk on the "files changed" tab?
An alternative, besides not mentioning it, is to present a link from GitHub on how to use the suggestions feature: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

Reply:
I like the idea of linking to the GitHub documentation. :+1: